base/libgit2/types.jl

```diff
@@ -554,7 +562,10 @@ function with_warn{T}(f::Function, ::Type{T}, args...)
     end
 end
 
-
+"""
+Convert between the julia Type of a git object and
```

Comment:
be consistent about capitalizing Julia type? lower case refers to the executable, upper case the language